README.md

```diff
@@ -54,61 +78,101 @@ CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
 CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api
 ```
 
-Note that if you are connecting as a host, the login should be prefixed with `host/`. For example: `host/myhost.example.com`,
-not just `myhost.example.com`.
+_NOTE:_ Defining environment variables is dependent on where the application is running. For example, if the Java app is running in Docker, you would set them in the `docker-compose.yml`.
```

Comment:
I don't think this line is needed, and you can drop it.

Not all Docker users will be using docker compose, either.